tikv/raftstore/peer_storage.go

```diff
@@ -291,6 +291,37 @@ func getMsg(engine *badger.DB, key []byte, msg proto.Message) error {
 	return proto.Unmarshal(val, msg)
 }
 
+type storageError string
+
+func (e storageError) Error() string {
+	return string(e)
+}
+
+func getRegionLocalStateMsg(db *badger.DB, regionId uint64) (*rspb.RegionLocalState, error) {
+	regionLocalState := new(rspb.RegionLocalState)
+	if err := getMsg(db, RegionStateKey(regionId), regionLocalState); err != nil {
+		return nil, &ErrRegionNotFound{regionId}
+	}
+	return regionLocalState, nil
+}
+
+func getRaftApplyStateMsg(db *badger.DB, regionId uint64) (*rspb.RaftApplyState, error) {
```

Comment:
`RaftApplyState` has been replaced by `raftState` in `state.go`.

Reply:
yes